alerts/src/androidLibMain/kotlin/Alerts.kt

```diff
@@ -33,6 +29,10 @@ actual class AlertInterface(
     private val context: Context
 ) : BaseAlertPresenter(alert) {
 
+    private inline fun <T> T.applyIf(condition: Boolean, block: T.() -> Unit): T = apply {
```

Comment:
We would also only have this ones if we put it in a good spot.

Reply:
If I put it in Components, then Alerts has to bring all dependencies which we have in Components. So it's better to make for example Utils component in dependency free stuff?

Reply:
Yes, `Components` is from before the split into modules. It contains a bunch of utils stuff bit needs to be split, see #16 You can pick it up if you want, till then I'd say it's ok for your module to depend on it.

Reply:
I tried to fulfil all dependencies but end up with circular one. I'd like to leave it here for now, so Alerts will be released and puck up #16 as next one